quickwit-storage/Cargo.toml

```diff
@@ -24,6 +24,8 @@ thiserror = '1'
 rand = '0.8'
 lru = "0.6"
 serde = { version = "1.0", features = ["derive"] }
+# hyper = { version = "0.14", features = ["client", "http1", "http2", "tcp"] }
+hyper = "0.14"
```

Comment:
Isn't adding `hyper` as a dependency for importing the constants of two widely known HTTP status codes a bit overkill?

Reply:
Okay, I’ll remove `hyper`.